src/index.html

```diff
@@ -6,9 +6,44 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200&family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu__head">
+      <a href="#" class="menu_logo">
+        <img src="./images/logo.png" alt="moyo logo">
+      </a>
+      <nav class="nav">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML. It's hard to read now. Check it everywhere 
```suggestion
      </a>
      
      <nav class="nav">
```

Reply:
done